src/CMakeLists.txt

```diff
@@ -78,6 +81,9 @@ elseif (CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
                 gles/pxClipboardNative.cpp gles/pxEGLProviderRPi.cpp gles/LinuxInputEventDispatcher.cpp pxViewWindow.cpp
                 unix/rtMutexNative.cpp unix/rtThreadPoolNative.cpp
                 rtFileCache.cpp rtHttpCache.cpp)
+         if (SUPPORT_DUKTAPE)
+          set(PXCORE_FILES ${PXCORE_FILES} rtScriptDuk/debugger/duk_debugger_interface_unix.c)
```

Comment:
Please update to the new name and location, rtScriptDuk/debugger/unix/duk_debugger_interface.c.